lib/eex/test/eex/tokenizer_test.exs

```diff
@@ -13,148 +13,176 @@ defmodule EEx.TokenizerTest do
-           {:ok, [{:text, 'foo '}, {:expr, 1, '=', ' bar '}]}
+    assert T.tokenize('foo <%= bar %>', 1) == {:ok, [{:text, 'foo '}, {:expr, 1, '=', ' bar '}]}
   end
 
   test "strings with embedded slash code" do
-    assert T.tokenize('foo <%/ bar %>', 1) ==
-           {:ok, [{:text, 'foo '}, {:expr, 1, '/', ' bar '}]}
+    assert T.tokenize('foo <%/ bar %>', 1) == {:ok, [{:text, 'foo '}, {:expr, 1, '/', ' bar '}]}
   end
 
   test "strings with embedded pipe code" do
-    assert T.tokenize('foo <%| bar %>', 1) ==
-           {:ok, [{:text, 'foo '}, {:expr, 1, '|', ' bar '}]}
+    assert T.tokenize('foo <%| bar %>', 1) == {:ok, [{:text, 'foo '}, {:expr, 1, '|', ' bar '}]}
   end
 
   test "strings with more than one line" do
-    assert T.tokenize('foo\n<%= bar %>', 1) ==
-           {:ok, [{:text, 'foo\n'}, {:expr, 2, '=', ' bar '}]}
+    assert T.tokenize('foo\n<%= bar %>', 1) == {:ok, [{:text, 'foo\n'}, {:expr, 2, '=', ' bar '}]}
   end
 
   test "strings with more than one line and expression with more than one line" do
     string = '''
-foo <%= bar
+    foo <%= bar
 
-baz %>
-<% foo %>
-'''
+    baz %>
+    <% foo %>
+    '''
 
-    assert T.tokenize(string, 1) == {:ok, [
-      {:text, 'foo '},
-      {:expr, 1, '=', ' bar\n\nbaz '},
-      {:text, '\n'},
-      {:expr, 4, '', ' foo '},
-      {:text, '\n'}
-    ]}
+    assert T.tokenize(string, 1) == {
+             :ok,
+             [
+               {:text, 'foo '},
+               {:expr, 1, '=', ' bar\n\nbaz '},
+               {:text, '\n'},
+               {:expr, 4, '', ' foo '},
+               {:text, '\n'}
+             ]
+           }
   end
 
   test "quotation" do
-    assert T.tokenize('foo <%% true %>', 1) == {:ok, [
-      {:text, 'foo <% true %>'}
-    ]}
+    assert T.tokenize('foo <%% true %>', 1) == {:ok, [{:text, 'foo <% true %>'}]}
   end
 
   test "quotation with do/end" do
-    assert T.tokenize('foo <%% true do %>bar<%% end %>', 1) == {:ok, [
-      {:text, 'foo <% true do %>bar<% end %>'}
-    ]}
+    assert T.tokenize('foo <%% true do %>bar<%% end %>', 1) == {:ok, [{:text, 'foo <% true do %>bar<% end %>'}]}
   end
 
   test "quotation with interpolation" do
-    assert T.tokenize('a <%% b <%= c %> <%= d %> e %> f', 1) == {:ok, [
-      {:text, 'a <% b '},
-      {:expr, 1, '=', ' c '},
-      {:text, ' '},
-      {:expr, 1, '=', ' d '},
-      {:text, ' e %> f'}
-    ]}
-
-    assert T.tokenize('<%%% a <%%= b %> c %>', 1) == {:ok, [
-      {:text, '<%% a <%= b %> c %>'}
-    ]}
+    assert T.tokenize('a <%% b <%= c %> <%= d %> e %> f', 1) == {
+             :ok,
+             [
+               {:text, 'a <% b '},
+               {:expr, 1, '=', ' c '},
+               {:text, ' '},
+               {:expr, 1, '=', ' d '},
+               {:text, ' e %> f'}
+             ]
+           }
+
+    assert T.tokenize('<%%% a <%%= b %> c %>', 1) == {
+             :ok,
+             [
+               {:text, '<%% a <%= b %> c %>'}
+             ]
+           }
```

Comment:
This one (and the ones that return a single expression only) you can probably rewrite it to this:

```elixir
 assert T.tokenize('<%%% a <%%= b %> c %>', 1) ==
         {:ok, [{:text, '<%% a <%= b %> c %>'}]}
```

Don't forget to format again after all your changes.